.github/ISSUE_TEMPLATE/new_python_tool_pip.yml

```diff
@@ -46,6 +46,13 @@ body:
       placeholder: ex. 1.4.11
     validations:
       required: true
+  - type: input
```

Comment:
[nitpick] I think the arguments should be placed just after `Tool's description` and before `Dependencies`. It feels for me more natural to first fill the required fields like `Category`.